### ISSUE TITLE
Culling ignores scaling.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [paazmaya/papervision3d#136]
```
What steps will reproduce the problem?
1. Create plane and position it so that it is culled.
2. set plane scale so that it should be visible.

Plane should be visible but it is culled.

I'm running it on FlashPlayer10 p3d rev#759.

in attach there is code example that reproduces the issue

That bug was raised here in comments
http://code.google.com/p/papervision3d/issues/detail?id=108&can=1&q=cull
but was not fixed.
```

Original issue reported on code.google.com by `maks...@gmail.com` on 24 Nov 2008 at 8:00

Attachments:
- [p3dCullerBug.rar](https://storage.googleapis.com/google-code-attachments/papervision3d/issue-136/comment-0/p3dCullerBug.rar)

**Comment by GoogleCodeExporter** [comment]
```
tim ? 
```

Original comment by `r.hauw...@gmail.com` on 1 Dec 2008 at 4:03

**Comment by GoogleCodeExporter** [comment]
```
fixed in rev #813. FrustumCuller did not account for do3d's scale when calcing 
the
object's bouding-sphere's radius
```

Original comment by `tim.k...@gmail.com` on 1 Dec 2008 at 6:22
- Changed state: **Fixed**
